jib-core/src/test/java/com/google/cloud/tools/jib/filesystem/AbsoluteUnixPathTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.filesystem;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/** Test for {@link AbsoluteUnixPath}. */
+public class AbsoluteUnixPathTest {
```

Comment:
I included those in `UnixPathParserTest`. I think we don't have to take care of `..` and `.` since those are valid unix path components and just have this class be absolute unix paths with no canonicalization guarantees.